.github/workflows/on_push.yml

```diff
@@ -96,12 +94,19 @@ jobs:
         run: python setup.py build_ext --inplace
       - name: "Add distribution info"  #  This lets SQLAlchemy find entry points
         run: python setup.py develop
+      - name: Add ClickHouse TLS instance to /etc/hosts
+        run: |
+          sudo echo "127.0.0.1 server1.clickhouse.test" | sudo tee -a /etc/hosts
       - name: Run tests
         env:
           CLICKHOUSE_CONNECT_TEST_DOCKER: 'False'
           CLICKHOUSE_CONNECT_TEST_FUZZ: 50
           SQLALCHEMY_SILENCE_UBER_WARNING: 1
         run: pytest tests
+      - name: Run TLS tests
+        env:
+          CLICKHOUSE_CONNECT_TEST_TLS: 1
+        run: pytest tests/tls
```

Comment:
This was previously missing. Now, it is possible, as we have a TLS instance in Docker.